bigquery_etl/static/__init__.py

```diff
@@ -1,36 +1,66 @@
-"""Publish csv files as BigQuery tables."""
```

Comment:
Pet peeve/nit: not a fan of having logic in `__init__.py` files 😝 would prefer a `static.py` file in this case or another file with a 1-line import here